### PR TITLE
Add more detailed system log messages

### PR DESCRIPTION
**Pull request by fritzmg**
This adds more detailed system log messages for the form subscription implementation.

### CODE REVIEW

File: src/EventListener/FormSubscriptionListener.php [hunk 1]
```diff
@@ -140,7 +140,19 @@ public function onProcessFormData(array $submittedData, array $formData, ?array
         if ($result) {
             $this->logger->log(LogLevel::INFO, sprintf('Successfully subscribed "%s" to Mailchimp list "%s".', $email, $model->listName), ['contao' => new ContaoContext(__METHOD__, TL_GENERAL)]);
         } else {
-            $this->logger->log(LogLevel::INFO, sprintf('Could not subscribe "%s" to Mailchimp list "%s".', $email, $model->listName), ['contao' => new ContaoContext(__METHOD__, TL_ERROR)]);
+            $msg = sprintf('Could not subscribe "%s" to Mailchimp list "%s".', $email, $model->listName);
+
+            if ($lastError = $api->getLastError()) {
+                if ($lastError->detail ?? null) {
```
Comment (bytehead) [inline src/EventListener/FormSubscriptionListener.php:146]: Don't we have to raise the minimum PHP version for this?

Comment (fritzmg) [reply src/EventListener/FormSubscriptionListener.php:146]: May be. Which PHP version introduced the null coalescing operator? 😁

Comment (bytehead) [reply src/EventListener/FormSubscriptionListener.php:146]: Hmm, could work :) https://www.php.net/manual/en/migration70.new-features.php looks like PHP 7.0 :)

**Review by bytehead — APPROVED** [review]

**Comment by bytehead** [comment]
Thank you ❤️ [4.5.0](https://github.com/1up-lab/contao-mailchimp/releases/tag/4.5.0) is released!

**Pull request merged**